src/views/repository/AnalyzeText.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <repository-view-base
+    :repository="repository"
+    :loading="loading"
+    :error-code="errorCode">
+    <div v-if="repository">
+      <div
+        v-if="authenticated"
+        class="repository-analyze-text" >
+        <div class="bh-grid">
+          <div class="bh-grid__item">
+            <div class="repository-analyze-text__item">
+              <p>Make a HTTP request to NLP service, follow the example bellow.</p>
+            </div>
+            <div class="repository-analyze-text__item">
+              <p><strong>URL:</strong></p>
+              <div class="pre">https://nlp.bothub.it/parse/</div>
+            </div>
+            <div class="repository-analyze-text__item">
+              <p><strong>Header:</strong></p>
+              <div class="pre">Authorization: Bearer {{ repository.authorization.uuid }} </div>
+            </div>
+            <div class="repository-analyze-text__item">
+              <p><strong>POST with form-data:</strong></p>
+              <pre class="pre">language: [language code]
+text: [text to analyze]</pre>
+            </div>
+            <div class="repository-analyze-text__item">
+              <p><strong>Response:</strong></p>
+              <pre class="pre">{
+  "intent": {
+    "name": "love",
+    "confidence": 0.6943462393863934
+  },
+  "intent_ranking": [
+    {
+      "name": "love",
+      "confidence": 0.6943462393863934
+    },
+    {
+      "name": "hate",
+      "confidence": 0.30565376061360666
+    }
+  ],
+  "labels_list": [
+    "animal"
+  ],
+  "entities_list": [],
+  "entities": {
+    "animal": [
+      {
+        "value": "puppy",
+        "entity": "dog",
+        "confidence": 0.67255946125065845
+      }
+    ],
+    "other": []
+  },
+  "text": "i love my puppy",
+  "update_id": 47,
+  "language": "en"
+}
+              </pre>
+            </div>
+          </div>
+          <div class="bh-grid__item">
+            <div class="repository-analyze-text__item">
+              <div><strong>Code Generator:</strong></div>
+              <div>Generate code to your respective programming language.</div>
+            </div>
+            <request-generator
+              :default-language-field="repository.language"
+              authorization-uuid="repository.authorizatiom.uuid"/>
```

Comment:
This works `repository.authorizatiom.uuid`? `authorizatiom`?
Try use `authorization`